view/adminhtml/templates/catalog/product/attribute/options.phtml

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/** @var $block \Magento\Eav\Block\Adminhtml\Attribute\Edit\Options\Options */
+
+$stores = $block->getStoresSortedBySortOrder();
```

Comment:
So this template might have changed between the versions of Magento this module supports. You'll need to look at the current `composer.json` and review Magento's official repo's and compare between the versions. If this template has changed between `"magento/module-eav": "^100|^101|^102",` (you may have to investigate and see which EAV module versions correspond to Magento versions) then we'd need to remove support for anything older than that and potentially tag this as a major version.

You may have to review in a similar fashion for `Magento\Eav\Block\Adminhtml\Attribute\Edit\Options\Options`

Reply:
On the ticket I also posited this 

>I am not sure in the difficulty of modifying this part of the MAP, if modifying this grid is very difficult or complex (and likely to break between different versions of Magento) perhaps we can add a new tab which shows a read only table of data with the option_code included?

If it seems that keeping this template in a way that works in 2.3.x and 2.4.x could be difficult would it be less difficult to add a new tab with a wholly new template that can be viewed on this page?

Reply:
just verified, no major changes in both files